src/sentry/dynamic_sampling/tasks/helpers/boost_low_volume_projects.py

```diff
@@ -16,7 +16,9 @@ def get_boost_low_volume_projects_sample_rate(
     cache_key = generate_boost_low_volume_projects_cache_key(org_id=org_id)
 
     try:
-        return float(redis_client.hget(cache_key, project_id)), True
+        val = redis_client.hget(name=cache_key, key=str(project_id))
+        assert val is not None
```

Comment:
I think this might be wrong. Previously we were looking for `TypeError` but `RedisCluster` and `StrictRedis` types suggests that `hget` returns `str | None`